src/ocamlorg_frontend/pages/package_overview.eml

```diff
@@ -66,6 +64,15 @@ Package_layout.render
             <br />
             <% ); %>
         </div>
+        <% (match documentation_status with
+            | `Success -> %>
+            <div class="mt-4"><a class="btn btn-sm" href="<%s Url.package_doc package.name package.version ~page:"index.html" %>">Documentation</a></div>
+            <% | `Unknown -> ( %><a href="<%s Url.package_doc package.name package.version ~page:"index.html" %>" class="p-4 mt-4 flex gap-x-2 bg-error-600 text-white font-semibold text-base">
+            <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="white"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M15.32 3H8.68c-.26 0-.52.11-.7.29L3.29 7.98c-.18.18-.29.44-.29.7v6.63c0 .27.11.52.29.71l4.68 4.68c.19.19.45.3.71.3h6.63c.27 0 .52-.11.71-.29l4.68-4.68c.19-.19.29-.44.29-.71V8.68c0-.27-.11-.52-.29-.71l-4.68-4.68c-.18-.18-.44-.29-.7-.29zM12 17.3c-.72 0-1.3-.58-1.3-1.3s.58-1.3 1.3-1.3 1.3.58 1.3 1.3-.58 1.3-1.3 1.3zm0-4.3c-.55 0-1-.45-1-1V8c0-.55.45-1 1-1s1 .45 1 1v4c0 .55-.45 1-1 1z"/></svg>
+            Documentation status is unknown. Sorry about that.</a><% )
+            | `Failure -> ( %><a href="<%s Url.package_doc package.name package.version ~page:"index.html" %>" class="p-4 mt-4 flex gap-x-2 bg-error-600 text-white font-semibold text-base">
+            <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="white"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M15.32 3H8.68c-.26 0-.52.11-.7.29L3.29 7.98c-.18.18-.29.44-.29.7v6.63c0 .27.11.52.29.71l4.68 4.68c.19.19.45.3.71.3h6.63c.27 0 .52-.11.71-.29l4.68-4.68c.19-.19.29-.44.29-.71V8.68c0-.27-.11-.52-.29-.71l-4.68-4.68c-.18-.18-.44-.29-.7-.29zM12 17.3c-.72 0-1.3-.58-1.3-1.3s.58-1.3 1.3-1.3 1.3.58 1.3 1.3-.58 1.3-1.3 1.3zm0-4.3c-.55 0-1-.45-1-1V8c0-.55.45-1 1-1s1 .45 1 1v4c0 .55-.45 1-1 1z"/></svg>
+            Documentation failed to build! Sorry about that.</a><% ));%>
```

Comment:
I'd rather go for something like this:
```html
        <div class="mt-4">
            <% (match documentation_status with
                | `Success -> %>
                <a class="btn btn-secondary" href="<%s Url.package_doc package.name package.version ~page:"index.html" %>">Documentation</a>
            <% | `Unknown ->  %>
                <div class="btn btn-sm-disabled" href="<%s Url.package_doc package.name package.version ~page:"index.html" %>">Documentation status is unknown. Sorry about that.</div>
            <% | `Failure -> %>
                <div class="btn btn-sm-disabled" href="<%s Url.package_doc package.name package.version ~page:"index.html" %>">Documentation failed to build! Sorry about that.</div>
            <%  ); %>
        </div>
```
* Slightly more compact
* No broken links
* More consistent appearance (rounded corners)

Reply:
Did I tell anyone I'm a bit fed up with _orange_ I mean as a color of course :-)

Reply:
Button color is about visual hierarchy. It's about guiding the eye of the user to the most important parts of the page. So making one of the most important links on the page a flashy Call-to-Action is reasonable. (We do want to make sure the user finds the documentation immediately, as this is most likely the reason they're browsing in the first place.)

I agree that the shade of orange we currently have is problematic because it does not meet the WCAG accessibility standards wrt to contrast.